website/content/en/preview/AWS/provisioning.md

```diff
@@ -23,28 +22,34 @@ spec:
   limits:
     resources:
       cpu: 1000                               # optional, recommended to limit total provisioned CPUs
-      memory: 1000Gi                          # optional, recommended to limit total provisioned memory
-  provider:
-    subnetSelector:                           # required
-      karpenter.sh/discovery: ${CLUSTER_NAME}
-    securityGroupSelector:                    # required, when not using launchTemplate
-      karpenter.sh/discovery: ${CLUSTER_NAME}
-    instanceProfile: MyInstanceProfile        # optional, if already set in controller args
-    launchTemplate: MyLaunchTemplate          # optional, see Launch Template documentation
-    tags:
-      InternalAccountingTag: 1234             # optional, add tags for your own use
+      memory: 1000Gi
+  providerRef:                                # optional, recommended to use instead of `provider`
+    name: default
   ttlSecondsAfterEmpty: 30                    # optional, but never scales down if not set
   ttlSecondsUntilExpired: 2592000             # optional, but never expires if not set
+---
+apiVersion: karpenter.k8s.aws/v1alpha1
+kind: AWSNodeTemplate
+metadata:
+  name: default
+spec:
+  subnetSelector:                             # required
+    karpenter.sh/discovery: ${CLUSTER_NAME}
+  securityGroupSelector:                      # required, when not using launchTemplate
+    karpenter.sh/discovery: ${CLUSTER_NAME}
+  instanceProfile: MyInstanceProfile          # optional, if already set in controller args
+  launchTemplate: MyLaunchTemplate            # optional, see Launch Template documentation
+  tags:
+    InternalAccountingTag: 1234               # optional, add tags for your own use

```

Comment:
Webhook seems to be complaining about this being an integer.  

```
Error from server (BadRequest): error when creating "providerref.yaml": admission webhook "defaulting.webhook.provisioners.karpenter.sh" denied the request: mutation failed: cannot decode incoming new object: json: cannot unmarshal number into Go struct field AWSNodeTemplateSpec.spec.tags of type string
```

Reply:
Ha! Good catch, I never tried this out. It's been around in our docs for a while